src/app/acesso/cadastro/cadastro.component.spec.ts

```diff
@@ -14,6 +14,9 @@ import { MatDialogModule } from '@angular/material/dialog';
 import { MatCardModule } from '@angular/material/card';
 import { CadastroPrestadorComponent } from './cadastro-prestador/cadastro-prestador.component';
 import { CadastroClienteComponent } from './cadastro-cliente/cadastro-cliente.component';
+import {EnderecoCadastroComponent} from './endereco-cadastro/endereco-cadastro.component'
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6334730)